beacon-light-client/solidity/contracts/validators_accumulator/ValidatorsAccumulator.sol

```diff
@@ -90,17 +89,12 @@ contract ValidatorsAccumulator is IValidatorsAccumulator {
       )
     );
 
-    bytes memory count = IDeposit(depositAddress).get_deposit_count();
-    emit Deposited(pubkey, signature, depositMessageRoot, count);
+    bytes memory depositIndex = IDeposit(depositAddress).get_deposit_count();
+    emit Deposited(pubkey, signature, depositMessageRoot, depositIndex);
```

Comment:
Maybe this should be in the same order as the it is in the node I think pubkey, depositIndex, signature, depositMessageRoot